templates/server-ha-active-service.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     app.kubernetes.io/name: {{ include "vault.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+    vault-active: "true"
```

Comment:
this is quite useful, resolving https://github.com/hashicorp/vault-helm/issues/445